.github/workflows/ci.yml

```
@@ -285,6 +87,19 @@ jobs:
         working-directory: ${{ env.UV_WORKSPACE }}
         run: rustup show
 
+      - name: "Reproduce error"
+        working-directory: ${{ env.UV_WORKSPACE }}
+        env:
+          # Avoid debug build stack overflows.
+          UV_STACK_SIZE: 2000000 # 2 megabyte, double the default on windows
+        run: |
+          git clone https://github.com/cda-tum/mqt-qcec
+          cd mqt-qcec
+          git switch use-mqt-core-package
+          nuget.exe install python-freethreaded -Version 3.13.0-rc1 -FallbackSource https://api.nuget.org/v3/index.json -OutputDirectory python-freethreaded.3.13.0-rc1
+          cargo run venv --python "python-freethreaded.3.13.0-rc1\tools\python.exe"
```

```suggestion
          nuget.exe install python-freethreaded -Version 3.13.0-rc1 -FallbackSource https://api.nuget.org/v3/index.json -OutputDirectory 'C:\Users\runneradmin\AppData\Local\test\Cache\nuget-cpython'
          cargo run venv --python "C:\Users\runneradmin\AppData\Local\test\Cache\nuget-cpython\python-freethreaded.3.13.0-rc1\tools\python.exe"
```
adapted from https://github.com/cda-tum/mqt-qcec/blob/use-mqt-core-package/.github/workflows/cd.yml